test/cache_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class TestOpenExchangeRates < Test::Unit::TestCase
+  def test_cache_key
+    cache = OpenExchangeRates::Cache::MemoryCache.new
+    assert_equal 'open-exchange-rates:2015-12-27:USD', cache.key(:from => 'USD', :on => '2015-12-27')
```

Comment:
Line is too long. [101/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.